.github/workflows/trigger_release_on_merge_to_main.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: Release new version on merge to main
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  create_release:
+    name: Release
+    uses: ./.github/workflows/create_release.yml
+    with:
+      # let the version be computed
+      version: unset
+      branch: main
+    secrets:
+      REPO_ACCESS_PAT: ${{ secrets.OASYS_REPO_SCOPE_GITHUB_PAT }}
```

Comment:
These two workflows should work, assuming we use a PAT here that has access to this repo (only), @renefritze?

Reply:
yeah